src/js/actions/index.js

```diff
@@ -26,6 +26,7 @@ import fileSaver from 'file-saver'
 
 let latestSubmissionId
 let currentFilingPeriod
+const pollObj = {polling: false}
```

Comment:
So that `pollForProgress can be passed a value that can be altered externally. Using an object means you can:
 - in the tests for `pollForProgress` pass in the value to check the behavior for each condition
 - change the polling value externally (ie from `setFile` or `setNewFile`

If it was just a boolean, you couldn't meet both these conditions simultaneously